README.md

```diff
@@ -1,7 +1,7 @@
 # Web Technologies Syllabus
 
-* **Course:** CPTR 335 - Web Technologies
-* **Instructor:** Benjamin Barber, [barberboy@gmail.com](mailto:barberboy@gmail.com)
+* **Course:** CPTR 305 - Web Technologies
```

Comment:
Sorry, this should be 335. I'm a **little** offended that you changed it to a 305.

:-1: